src/heitech.zer0mqXt.core/Main/Socket.cs

```diff
@@ -72,5 +76,32 @@ public void RegisterSubscriber<TMessage>(Action<TMessage> callback, Cancellation
             if (result.IsSuccess == false)
                 throw result.Exception;
         }
+
+        public async Task SendAsync<TMessage>(TMessage message) 
```

Comment:
Should we put [inheritdoc](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/inheritdoc) for those method implementations?